plugins.yaml

```diff
@@ -2,12 +2,14 @@ plugins:
   bartoszmajsak/wfswarm-booster-pipeline-test:
   - size
 
+# tag::externalPlugins[]
 external_plugins:
   bartoszmajsak/wfswarm-booster-pipeline-test:
```

Comment:
when this file is included in the README, then it would be good to change it to something general: 
`organization/repository`

Reply:
The reason why I include this file is to have it also testable, for that purpose it references to an actual existing repository. We can change it in the future, but I would still prefer to have it as "living" config. Otherwise, we can simply just revert it to be just a text in the doc.

Reply:
make sense, keep it as it is

Reply:
But yeah, don't want to have it as my name everywhere. Let's change this and tests soon.